<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Apackager: MPKG Package manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/applications-system.png</normaloff>:/icons/applications-system.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBoxDistr"/>
        </item>
        <item>
         <widget class="QListView" name="tagView">
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButtonCategory">
            <property name="text">
             <string>Category</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonTags">
            <property name="text">
             <string>Tags</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="pkgView">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="cmdChkAll">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/installed.png</normaloff>:/icons/installed.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="cmdUchkAll">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="cmdResetAll">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxFilter"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditFilter"/>
          </item>
          <item>
           <widget class="QToolButton" name="cmdClearFilter">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdCommit">
            <property name="text">
             <string>Commit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMPKG">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuRopositories">
    <property name="title">
     <string>MPKG</string>
    </property>
    <addaction name="actionCommit"/>
    <addaction name="separator"/>
    <addaction name="actionUpdate_repository_data"/>
    <addaction name="actionReload_package_list"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionReset_queue"/>
    <addaction name="actionClean_cache"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
    <addaction name="actionHide_deprecated"/>
    <addaction name="actionHide_installed"/>
    <addaction name="actionHide_not_installed"/>
    <addaction name="actionHide_installed_deprecated"/>
    <addaction name="actionHide_updated"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPackage_info"/>
    <addaction name="actionDependency"/>
    <addaction name="separator"/>
    <addaction name="actionToolbar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMPKG"/>
   <addaction name="menuRopositories"/>
   <addaction name="menuView"/>
   <addaction name="menuFilters"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetPackageInfo">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Package info</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextEdit" name="textEditPkgInfo">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="overwriteMode">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetDependency">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Dependencies info</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QTextEdit" name="textEditDependencies">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionPackage_info">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Package info</string>
   </property>
  </action>
  <action name="actionDependency">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dependency</string>
   </property>
  </action>
  <action name="actionHide_deprecated">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide deprecated</string>
   </property>
  </action>
  <action name="actionHide_installed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide installed</string>
   </property>
  </action>
  <action name="actionHide_not_installed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide not installed</string>
   </property>
  </action>
  <action name="actionHide_installed_deprecated">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide installed deprecated</string>
   </property>
  </action>
  <action name="actionHide_updated">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide updated</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/available-file.png</normaloff>:/icons/available-file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCommit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/run.png</normaloff>:/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </action>
  <action name="actionUpdate_repository_data">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Update repository data</string>
   </property>
   <property name="toolTip">
    <string>Update repository data</string>
   </property>
  </action>
  <action name="actionReload_package_list">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reload package list</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionReset_queue">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/trash-empty.png</normaloff>:/icons/trash-empty.png</iconset>
   </property>
   <property name="text">
    <string>Reset queue</string>
   </property>
  </action>
  <action name="actionClean_cache">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-clear.png</normaloff>:/icons/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clean cache</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <zorder>dockWidgetPackageInfo</zorder>
  <zorder>dockWidgetDependency</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
